frontend_app_html-document/src/component/HtmlDocument.jsx

```diff
@@ -28,7 +31,18 @@ export const HtmlDocument = props => {
 
   useEffect(() => {
     setText(props.text)
-  }, [])
+  }, [props.text])
```

Comment:
could you rename the state "text" to be more clear that this state is the text that user will update and props.text is the currently saved version?

I'm saying this because, in the render, you sometimes pass props.text and sometimes state text and it confused me a bit